cmd/fyne/internal/commands/build.go

```diff
@@ -53,8 +53,7 @@ func Build() *cli.Command {
 				Destination: &b.release,
 			},
 			&cli.StringFlag{
-				Name:        "executable",
-				Aliases:     []string{"exe", "o"},
+				Name:        "o",
```

Comment:
I think these docs were copied from the old "executable" input parameter flag, so it's a little strange.
The -o which should go to all commands later, needs docs that will make sense. Perhaps:
"Specify a name for the output file, default is based on the current directory".